flake.lock

```diff
@@ -41,11 +41,11 @@
     },
     "nixpkgs": {
       "locked": {
-        "lastModified": 1704538339,
-        "narHash": "sha256-1734d3mQuux9ySvwf6axRWZRBhtcZA9Q8eftD6EZg6U=",
+        "lastModified": 1704194953,
+        "narHash": "sha256-RtDKd8Mynhe5CFnVT8s0/0yqtWFMM9LmCzXv/YKxnq4=",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "46ae0210ce163b3cba6c7da08840c1d63de9c701",
+        "rev": "bd645e8668ec6612439a9ee7e71f7eac4099d4f6",
```

Comment:
Because of failing alacritty 0.12.3 => 0.13.0

https://github.com/kachick/dotfiles/actions/runs/7473234080/job/20337011480?pr=380

Reply:
https://github.com/NixOS/nixpkgs/issues/279778 => https://github.com/NixOS/nixpkgs/pull/279907